src/sentry/notifications/utils/participants.py

```diff
@@ -230,12 +234,35 @@ def determine_eligible_recipients(
         if team:
             return {team}
 
+    elif target_type == ActionTargetType.RELEASE_MEMBERS:
+        return get_release_committers(project, event)
+
     else:
         return get_owners(project, event)
 
     return set()
 
 
+def get_release_committers(project: Project, event: Event) -> Sequence[User]:
+    # get_participants_for_release seems to be the method called when deployments happen
+    # supposedly, this logic should be fairly, close ...
+    # why is get_participants_for_release so much more complex???
+    if not project or not event:
+        return []
+
+    last_release: Release = event.group.get_last_release()
```

Comment:
I think it's ok to use the latest release from the `group` for now, but later on we might want to find the specific release from the event and fetch if like `Release.objects.get(organization_id=project.organization_id, version=event_version)`.